packages/header-change-detection/lib/lambda/header-check.ts

```diff
@@ -0,0 +1,300 @@
+import axios from "axios";
+import { DynamoDBClient, BatchGetItemCommand, BatchGetItemCommandInput, KeysAndAttributes, UpdateItemCommandInput, UpdateItemCommand, AttributeValueUpdate } from "@aws-sdk/client-dynamodb";
```

Comment:
Surprised eslint or prettier didn't split this up.

Reply:
Yeah agree :thinking: but it doesn't seem to mind